samples/client/petstore-security-test/python/petstore_api/api_client.py

```diff
@@ -1,44 +1,32 @@
 # coding: utf-8
-
 """
-Copyright 2016 SmartBear Software
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
+    Swagger Petstore */ ' \" =end -- \\r\\n \\n \\r
```

Comment:
@frol it's due to license removal and additional security test case. No worry :)